synapse/rest/client/v1/admin.py

```diff
@@ -175,6 +175,7 @@ def on_POST(self, request):
         from synapse.rest.client.v2_alpha.register import RegisterRestServlet
         register = RegisterRestServlet(self.hs)
 
+        password = password.decode('utf-8')
```

Comment:
if there's a reason for doing `password.decode('utf-8')` rather than `body["password"]`, can you add a comment to say what it is?